WordPress/src/main/java/org/wordpress/android/ui/DeepLinkingIntentReceiverActivity.java

```diff
@@ -36,8 +40,28 @@ protected void onCreate(Bundle savedInstanceState) {
 
         // check if this intent is started via custom scheme link
         if (Intent.ACTION_VIEW.equals(action) && uri != null) {
-            mBlogId = uri.getQueryParameter("blogId");
-            mPostId = uri.getQueryParameter("postId");
+
+            switch (uri.getScheme()) {
+                case "wordpress":
+                    mBlogId = uri.getQueryParameter("blogId");
+                    mPostId = uri.getQueryParameter("postId");
+                    break;
+                case "http":
+                case "https":
+                    List<String> segments = uri.getPathSegments();
+
+                    if (segments != null && segments.get(0).equals("read")) {
```

Comment:
A comment here would be helpful. Nothing fancy, it could just be an example of a URL that would be handled here.